blockchain.go

```diff
@@ -207,6 +210,7 @@ var defaultConfig = func() config {
 		Logger:                       zerolog.Nop(),
 		TransactionValidationEnabled: true,
 		ChainID:                      flowgo.Emulator,
+		CoverageReportingEnabled:     true,
 	}
```

Comment:
I think we can get this from command line flag ( default can be true ) I am also totally fine with making it default on without any flag. but if we make config I think making it configurable would be better

Reply:
Yes, I definitely have to add a command line flag also. But I guess we need this one also, in case the `Blockchain` is utilized programmatically, for example, from another library. In this case the command line flag will not be sufficient I think. Correct me if I'm wrong though.

Reply:
true, you can expose it with a function like `WithCoverageReportingEnabled`  ( example: https://github.com/onflow/flow-emulator/blob/1fd1c61822d4771ac2dcad7a4f908bd0586ac293/blockchain.go#L368 ) 

then you need to add to `configureBlockchain`: https://github.com/onflow/flow-emulator/blob/1fd1c61822d4771ac2dcad7a4f908bd0586ac293/server/server.go#L335

then to this `config`: https://github.com/onflow/flow-emulator/blob/1fd1c61822d4771ac2dcad7a4f908bd0586ac293/server/server.go#L85

and finally: https://github.com/onflow/flow-emulator/blob/1fd1c61822d4771ac2dcad7a4f908bd0586ac293/cmd/emulator/start/start.go#L42

and 

https://github.com/onflow/flow-emulator/blob/1fd1c61822d4771ac2dcad7a4f908bd0586ac293/cmd/emulator/start/start.go#L161

Turtles all the way down :)

Reply:
Flags, flags everywhere :joy_cat: 
Updated in https://github.com/onflow/flow-emulator/pull/332/commits/8a6d81abc55a31654b1819e8391cec658e2e2af8